### PR TITLE
data:obaidashfaq20

### PR DESCRIPTION
**Pull request by obaidashfaq20**

### CODE REVIEW

**Review by dan-mba — DISMISSED** [review]
To fix the JSON error, add a comma at the end of line 6.

File: data/obaidashfaq20.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Obaid Ashfaq",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "a Ruby developer who is enthusiastic about writing clean code",
+  "avatar": "https://github.com/obaidashfaq20.png"
```
Comment (AbbeyIT) [inline data/obaidashfaq20.json:6]: Hi, please put a comma at the end of the line. So it won't have a syntax error in the json format.

**Review by AbbeyIT — CHANGES_REQUESTED** [review]

File: data/obaidashfaq20.json (hunk 1 above)
Comment (devvsakib) [inline data/obaidashfaq20.json:6]: you can add tags here, and add a comma to line 6

**Review by devvsakib — CHANGES_REQUESTED** [review]

File: data/obaidashfaq20.json (hunk 1 above)
Comment (SaraJaoude) [inline data/obaidashfaq20.json:6]: The avatar is no longer needed 

```suggestion
```

File: data/obaidashfaq20.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Obaid Ashfaq",
+  "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/obaidashfaq20.json:3]: Following a recent improvement this is no longer needed
```suggestion
```

**Pull request merged**